app/views/users/_additional_authentication.html.erb

```diff
@@ -2,10 +2,9 @@
   <div class="crayons-card crayons-card--content-rows block">
     <% authentication_enabled_providers.each do |provider| %>
       <!--
-        TODO: [@forem/oss] Only until we refactor our cookie configurations
-        we'll be able to connect existing accounts with SIWA. This should be
-        done when Apple Auth Provider is no longer in 'Beta':
-        https://github.com/forem/forem/pull/12114
+        INFO: [@forem/oss] We only support creating new accounts and then
+        signing in with them) using Apple Auth. Connecting Forem existing
+        accounts to Apple Auth isn't supported and is why we skip it here.
       -->
       <% next if provider.provider_name == :apple %>
```

Comment:
The option to connect existing accounts with Apple Accounts hasn't been requested (in the months DEV and other Forems have had SIWA enabled), the technical implications to do so continue to be non-trivial to implement + might result in security vulnerabilities, and a separate project that allows for cross-Forem Authentication capabilities will be worked on in the near future.

Because of the above we won't support this option. I added a spec in `spec/requests/user/user_settings_spec.rb` to ensure this is the case.